packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerSize.stories.tsx

```diff
@@ -52,3 +52,11 @@ export const SwatchPickerSize = () => {
     </div>
   );
 };
+
+SwatchPickerSize.parameters = {
+  docs: {
+    description: {
+      story: 'The `size` prop sets width and height of the Swatch. The default is `medium`.',
```

Comment:
Is there any reference somewhere to the differences between the sizes? If not, can that be added here?

Reply:
What differences do you mean?